modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/sorting/Sorting.tsx

```diff
@@ -132,17 +140,9 @@ const AddFDSSortModalContent = ({
 	const handleSave = async () => {
 		setSaveButtonDisabled(true);
 
-		const field = fields.find(
-			(item: IField) => item.name === selectedFieldName
-		);
-
-		if (!field) {
-			openDefaultFailureToast();
+		// Add new sorting.
```

Comment:
Ya that makes sense to only have them for the weird parts. I used them here as a way to divide the code up since the blocks were a bit long, but I can see it's not totally necessary.